docs/topics/move_big_data.md

```diff
@@ -0,0 +1,79 @@
+#<a name="top"></a>How to massively move data from/to the Platform (Big Data)
+Moving massive data from or to the IoT Platform means interacting with [Hadoop](http://hadoop.apache.org/)'s HDFS.
+
+HDFS stands for Hadoop Distributed File System. It is, in fact, a file system, since the data is organized as large data files following a Unix file system design. And it is distributed, since the large data files are not hosted by a single server but by a cluster of servers, having the files split into many pieces (blocks) all along the cluster.
+
+A special daemon, the NameNode, is in charge of managing HDFS, handling a table with file splits and their location within the data servers. This daemon talks with a set of DataNode's, i.e. the NameNode counterpart daemons running in the data servers.
+
+Doing I/O with HDFS is as <i>easy</i> as talking with the NameNode and asking for read and write operations, and file management in terms of ownership, permissions, etc. as well.
+
+There are several ways of interacting with a HDFS's Namenode. Let's see the ones supported by IoT Platform.
+
+##<a name="section1"></a>Methods
+###<a name="section1.1"></a>Using the FileSystem Shell
```

Comment:
Not sure this is a valid way to our customers/integrators. Note that this documentation is for integrators. It could be confusing.